src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOSmacOS.cs

```diff
@@ -239,6 +291,23 @@ private void OnRunJavaScriptTextInputPanel(WKWebView webview, string prompt, str
 
 			var controller = webview.FindViewController();
 			controller?.PresentViewController(alert, true, null);
+#else
+			var alert = new NSAlert()
+			{
+				AlertStyle = NSAlertStyle.Informational,
+				InformativeText = prompt,
+			};
+			var textField = new NSTextField()
```

Comment:
You will need to set a frame to your TextField else it will have a width and height of 0.
Something like: `var textField = new NSTextField(new CGRect(0, 0, 300, 20));`